docs/sqllab.rst

```diff
@@ -79,16 +79,11 @@ Superset's Jinja context:
 
 `Jinja's builtin filters <http://jinja.pocoo.org/docs/dev/templates/>`_ can be also be applied where needed.
 
-.. autofunction:: superset.jinja_context.ExtraCache.current_user_id
-
-.. autofunction:: superset.jinja_context.ExtraCache.current_username
-
-.. autofunction:: superset.jinja_context.ExtraCache.url_param
+.. autoclass:: superset.jinja_context.ExtraCache
```

Comment:
@villebro I thought it made more sense to generate documentation for all the members (which were explicitly enumerated previously).